includes/admin/class-wc-payments-admin-sections-overwrite.php

```diff
@@ -74,4 +83,28 @@ public function add_checkout_sections( array $sections ): array {
 
 		return $sections;
 	}
+
+	/**
+	 * Overwrites `$current_section`.
+	 *
+	 * Called before rendering the section list to render the "WooCommerce
+	 * Payments" section as active on payment method settings pages.
+	 */
+	public function overwrite_current_section_global() {
+		global $current_section;
+
+		$this->previous_current_section = $current_section;
+		if ( WC_Payments_Admin::is_payments_settings_page() ) {
```

Comment:
It looks like this is now creating a dependency between `WC_Payments_Admin` and `WC_Payments_Admin_Sections_Overwrite` - what do you think about moving the `is_payments_settings_page` static method to the `WC_Payments_Utils` class?

Reply:
Great catch!